app/presenters/planning_application_presenter.rb

```diff
@@ -23,6 +23,16 @@ def application_type_name
     I18n.t("application_types.#{application_type}")
   end
 
+  def application_type_abbreviation
+    I18n.t("application_types.#{application_type}_abbr", default: application_type_name)
+  end
+
+  def application_type_with_status
+    status_title = work_status.titlecase
+    "<abbr title=\"#{application_type_name} #{status_title}\">" \
+    "#{application_type_abbreviation} #{status_title}</abbr>".html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
Can we use `sanitize` instead here?

I tend to try and avoid `html_safe` if possible. 
You might need to `include ActionView::Helpers::SanitizeHelper` if the method isn't available.

Reply:
Perhaps I've misunderstood — I thought `html_safe` was necessary in order for the returned string to be displayed as HTML. If not `html_safe` then the HTML in this string would be escaped.

`sanitize` does almost the opposite, removing untrusted HTML from the string. Wouldn't it still need to be marked as safe in order to be rendered correctly?

And I do agree that it's good to avoid using `html_safe` — but in this case, everything that goes into making up the HTML is from a trusted source, I believe. And, because of the way the table view component simply loops over a list of the model's attributes, it's difficult to treat just one of the attributes differently at the view level. It would mean an awkward special case was needed to add the abbreviation here.